### PR TITLE
[BX-1103][BX-1296][BX-1298] Send NFTs

### PR DESCRIPTION
**Pull request by derHowie**
## What changed (plus any additional context for devs)
Added NFTs to the send screen and the send button to NFT details, added send nft e2e, and tweaked existing NFT UI e2e.

## Screen recordings / screenshots
PoW:
- UI: https://recordit.co/UbuqkDsk5O
- interaction with send token restoration + send: https://recordit.co/sEuYX3Ulcp

## What to test
Ensure sending NFTs works directly from the send screen and also when navigation from NFT Details.

### CODE REVIEW

**Review by DanielSinclair — APPROVED** [review]

File: src/core/references/index.ts [hunk 1]
```diff
@@ -51,6 +55,10 @@ export const smartContractMethods = {
     method: 'safeTransferFrom(address,address,uint256)',
     hash: '0x42842e0e',
   },
+  erc721_transfer_from: {
```
Comment (estebanmino) [inline src/core/references/index.ts:58]: open question, this is the same than erc1155 transfer?

File: src/entries/popup/hooks/send/useSendValidations.ts [hunk 2]
```diff
@@ -102,16 +117,16 @@ export const useSendValidations = ({
       }
     };
     checkToAddress();
-  }, [asset?.chainId, toAddress]);
+  }, [asset?.chainId, nft, toAddress]);
```
Comment (estebanmino) [inline src/entries/popup/hooks/send/useSendValidations.ts:120]: nft is a dep here, not sure if intended

File: src/core/utils/transactions.ts [hunk 3]
```diff
@@ -91,6 +92,72 @@ export const getDataForTokenTransfer = (value: string, to: string): string => {
...
+  from: string,
+  to: string,
+  asset: UniqueAsset,
+): string | undefined => {
+  if (!asset.id || !asset.asset_contract?.address) return;
+  const lowercasedContractAddress = asset.asset_contract.address.toLowerCase();
+  const standard = asset.asset_contract?.schema_name;
+  let data: string | undefined;
+  if (
+    lowercasedContractAddress === CRYPTO_KITTIES_NFT_ADDRESS &&
```
Comment (estebanmino) [inline src/core/utils/transactions.ts:122]: why ck is different?

File: src/core/utils/transactions.ts [hunk 4]
```diff
@@ -91,6 +92,72 @@ export const getDataForTokenTransfer = (value: string, to: string): string => {
...
+    asset.network === ChainName.mainnet
+  ) {
+    const transferMethod = smartContractMethods.punk_transfer;
+    data = getDataString(transferMethod.hash, [
+      removeHexPrefix(to),
+      convertStringToHex(asset.id),
+    ]);
+  } else if (standard === TokenStandard.ERC1155) {
+    const transferMethodHash =
+      smartContractMethods.erc1155_safe_transfer_from.hash;
```
Comment (estebanmino) [inline src/core/utils/transactions.ts:141]: ah i see the erc1155 method here

**Review by estebanmino — APPROVED** [review]
code lgtm

i didn't send any nft

**Pull request merged**